src/main/kotlin/dev/arbjerg/lavalink/client/LavalinkNodeOptions.kt

```diff
@@ -0,0 +1,39 @@
+package dev.arbjerg.lavalink.client
+
+import dev.arbjerg.lavalink.client.loadbalancing.IRegionFilter
+import dev.arbjerg.lavalink.internal.TIMEOUT_MS
+import java.net.URI
+
+data class LavalinkNodeOptions(val name: String,
+                               val serverUri: URI,
+                               val password: String,
+                               val regionFilter: IRegionFilter?,
+                               val httpTimeout: Long) {
+    data class Builder(
+        var name: String? = null,
```

Comment:
Why is everything but name private?

Reply:
at one point it was accessed when the name is checked for dupe nodes in the client, but I never privated it after changing it to check the built node options.